examples/trigger-example/src/example-trigger.ts

```diff
@@ -56,7 +56,7 @@ class IncrementDurationTriggerProvider implements TriggerProvider {
         // we have all the information we need in the `modelDelta`. But in the general case,
         // a trigger provider will need to find other related objects in the model and generate
         // new changes from those, so that's how we do it here
-        const model = await this.modelServerClient.get(modelURI.toString());
+        const model = await this.modelServerClient.get(modelURI);
```

Comment:
Do you mean like the REST interface when you talk about the `ModelService` API?
If so, why should we switch to using the `ModelService` API instead of the `ModelServerClient`?

Reply:
The [`ModelService`](https://github.com/eclipse-emfcloud/modelserver-node/blob/main/packages/modelserver-node/src/services/model-service.ts) offers a subset of `ModelServerClient` actions for a specific `modelUri` and directly delegates to it as far as i can see.
Currently we only use it in the dedicated `DefaultModelService` integration test right?
It is also not entirely clear to me why we should then switch to the service instead, do we gain any advantage there?

Reply:
The `ModelServerClient` just talks to the upstream Java server. So, it does not include any of the "business logic" defined in the Node server. That's what the `ModelService` is for, to let business logic plug-ins self-invoke the Node server, not the Java server, which they should always do. Only the Node server core framework should use the `ModelServerClient` to delegate to the Java server, unless maybe Node server plug-ins implementing custom REST endpoints also need to delegate those to the Java server.

So, I want the examples to show proper delegation of the REST APIs to from plug-ins to the Node server via `ModelService` instead of `ModelServerClient`. But it's not strictly in scope of this PR; I just noticed that we were making changes to stuff that uses the `ModelService` in the tests. I'll raise a separate issue for it.

Reply:
I created #60 for this, so nothing to do here. Sorry for the distraction!

Reply:
Thanks for the insights and the follow-up @cdamus!